generators/app/templates/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "<%= elementName %>",
+  "description": "<%= elementDescription %>",
+  "private": true,
+  "version": "0.0.0",
+  "license": "MIT",
+  "author": "<%= elementAuthor %>",
+  "repository": "",
+  "bugs": "",
+  "engines": {
+    "node": ">=10.0.0 <12"
+  },
+  "scripts": {
+    "clean": "rm -rf build",
+    "purge": "rm -rf node_modules",
+    "dev": "node_modules/@boringcodes/backpack/bin/backpack dev",
+    "prebuild": "yarn clean",
+    "build": "node_modules/@boringcodes/backpack/bin/backpack build",
```

Comment:
after installing backpack, npm add files to bin dir so you just simply run `backpack dev` or `backpack build`